Speed up snowflake profiler using approximation [sc-5603] (#117)
MetaphorData/connectors: metaphor/snowflake/profile/README.md

The config file inherits all the required and optional fields from the general Snowflake connector [Config File](../README.md#config-file).

### Optional Configurations

> #### Exclude_views

Reviewer: ```suggestion
#### Exclude_views
```
Exclude Views

By default, the connector only profile Snowflake "base table", i.e. exclude views and temporary tables. The following config can enable profiling on all tables and views.

```yaml
exclude_views: false
```

> #### Sample Probability

Reviewer: ```suggestion
#### Sampling
```

> For tables with large number of rows, profiling can be inefficient. To speed up the process, we can choose to do percentage based random sampling on the data by setting the config below:

Reviewer: ```suggestion
For tables with large number of rows, it can take a long time to profile. To speed up the process, we can choose to do a percentage-based random sampling on the data by setting `sample_probability`:
```

```yaml

> sample_percentage: <number between 0 and 100>

Reviewer: `sampling_percentage` might be a better name.

Author: updated

```

> For example, `sample_probability = 1` means random sampling of 1% rows in the table. Keep in mind that sampling won't apply to smaller tables (with less than 100K rows), in that case, we do complete table profiling.

Reviewer: Is "probability" the right word here? Or should we call it "ratio" or "percentage"?

Author: changed to percentage
